lib/nosedrum/storage.ex

```diff
@@ -1,75 +1,142 @@
 defmodule Nosedrum.Storage do
   @moduledoc """
-  Storages contain commands and are used by command invokers to look up commands.
+  `Storage`s keep track of your Application Command names and their associated modules. A `Storage` handles incoming
```

Comment:
To keep the summary in the ex doc overview nice, we should add an empty line between the first and second sentence.